panoramix/assets/visualizations/filter_box.js

```diff
@@ -10,15 +10,15 @@ function filterBox(slice) {
   var d3token = d3.select(slice.selector);
 
   var fltChanged = function() {
-    var filters = []
     for (var filter in filtersObj) {
       var obj = filtersObj[filter];
       var val = obj.val();
+      var vals = [];
       if (val !== '') {
-        filters.push([filter, val.split(',')]);
+        vals = val.split(',');
```

Comment:
I'm not sure if there's a better solution, but FYI @ interana we ran into huge issues trying to split strings this way once strings with ','s in them surfaced.